package-lock.json

```diff
@@ -6850,6 +6842,12 @@
             "p-limit": "^2.2.0"
           }
         },
+        "path-exists": {
```

Comment:
Is this a new transitive dependency? Also `paging-algorithm`

Reply:
I wouldn't worry about the changes in lower level deps in general. This one, for example, was already a transitive dep, it just changed where it was referenced. Paging-algorithm is a new algorithm used by our pagination package. As a rule, while it makes sense to sanity-check, we shouldn't get consumed by checking sub-deps, especially when we've read changelogs and tested.